pkgs/servers/irc/robustirc-bridge/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "robustirc-bridge";
+  version = "1.8";
+
+  src = fetchFromGitHub {
+    owner = "robustirc";
+    repo = "bridge";
+    rev = "v${version}";
+    sha256 = "12jzil97147f978shdgm6whz7699db0shh0c1fzgrjh512dw502c";
+  };
+
+  modSha256 = "0f93w3f1c23qkxx67kgqc0r3g2nkw1ngxx4rx946mv91h23vq596";
+
+  postInstall = ''
+    install -D $src/robustirc-bridge.1 $out/share/man/man1/robustirc-bridge.1
```

I just came across a [Discourse post](https://discourse.nixos.org/t/odd-uses-of-src-in-packages/6526/) suggesting that it should be better to avoid using `$src` directly like that. Could you test if you can access this file without `$src`? My guess is that this should work as well:

```suggestion
    install -D robustirc-bridge.1 $out/share/man/man1/robustirc-bridge.1
```

---

It works without $src. Thank you.